proto-app/src/components/Auth/Form/Form.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import Button from "../../UI/Button/Button";
+
+import classes from "./Form.module.scss";
+import iconUser from "../../../assets/user-icon.svg";
+import iconPass from "../../../assets/password-icon.svg";
+
+const Form = () => {
+	const [emailValue, setEmailValue] = useState("");
+	const [passValue, setPassValue] = useState("");
+
+	const submitHandler = (evt) => {
+		evt.preventDefault();
+		console.log("Submitted");
+	};
+	return (
+		<section className={classes.form}>
+			<form onSubmit={submitHandler}>
+				<div>
+					<label htmlFor="email" />
+					<input
+						type="email"
+						name="email"
+						placeholder="Email address"
+						style={{
+							backgroundImage: `url(${iconUser})`,
+						}}
+						value={emailValue}
+						onChange={(evt) => {
+							setEmailValue(evt.target.value);
+						}}
+					/>
+				</div>
+				<div>
+					<label htmlFor="password" />
+					<input
+						type="password"
+						name="password"
+						placeholder="Password"
+						style={{
+							backgroundImage: `url(${iconPass})`,
+						}}
+						value={passValue}
+						onChange={(evt) => {
+							setPassValue(evt.target.value);
+						}}
+					/>
+				</div>
+				<p className={classes.linkTerms}>
+					By logging in your agree to the &nbsp;
```

Comment:
you can see there is too much space between this line and the terms of use link

```suggestion
					By logging in your agree to the 
```